preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -410,9 +441,19 @@ def identity(
         if args and "type" in args:
             match args["type"]:
                 case "int":
-                    output_datum = int(input_datum)
+                    try:
+                        output_datum = int(input_datum)
+                    except ValueError:
+                        output_datum = None
+                        warnings.append(invalid_value_annotation(input_datum, output_field, "int"))
                 case "float":
-                    output_datum = float(input_datum)
+                    try:
+                        output_datum = float(input_datum)
+                    except ValueError:
+                        output_datum = None
+                        warnings.append(
+                            invalid_value_annotation(input_datum, output_field, "float")
```

Comment:
Do we definitely want this sort of thing to be just a warning not an error?

Reply:
I think the standard is just an error if the field is required (which will be the case), here we return a warning so the user will see this and then if they ignore it should be submitted without this field (e.g. output_datum=null)

Reply:
Tested and this works